openrtb_ext/request.go

```diff
@@ -76,6 +76,9 @@ type ExtRequestPrebid struct {
 	// the tracker URLs, where PBS will replace the Custom Macro with its value with url-encoding
 	Macros map[string]string `json:"macros,omitempty"`
 
+	// ORTBErrors enables OpenRTB-compliant error responses if set to true
+	ORTBErrors *bool `json:"ortberrors,omitempty"`
```

Comment:
If we only care about the true value, why does this need to be a pointer to differentiate between false and null?